cmd/utils/global_options.go

```diff
@@ -163,6 +131,128 @@ func DistributionAccountSignatureClientProviderOption(configKey *signing.Signatu
 	}
 }
 
+func RPCURLOption(configKey *string) *config.ConfigOption {
+	return &config.ConfigOption{
+		Name:        "rpc-url",
+		Usage:       "The URL of the RPC Server.",
+		OptType:     types.String,
+		ConfigKey:   configKey,
+		FlagDefault: "localhost:8080",
+		Required:    true,
+	}
+}
+
+func RPCCallerServiceChannelBufferSizeOption(configKey *int) *config.ConfigOption {
```

Comment:
Would these be considered "global options"? If they're only gonna be used in the `serve` command then I'd suggest we could move them to a separate file, `tss_options.go` perhaps? The `global_options.go` file was meant to hold the options that are shared across different commands.

Reply:
That makes sense. I moved these to tss_options.go